tests/integration/docusaurus/validation/validator/how_to_create_and_edit_expectations_with_instant_feedback_block_config.py

```diff
@@ -29,6 +29,7 @@
 """
 context.test_yaml_config(datasource_yaml)
 context.add_datasource(**yaml.load(datasource_yaml))
+context.add_or_update_expectation_suite(expectation_suite_name="my_expectation_suite")
```

Comment:
Was this necessary to get a test to pass? If so I think this is a bug that needs to be fixed - if it is needed, then this workflow would break if a user tried it.

Reply:
Things failed with "no suite named my_expectation_suite found" or something

Reply:
Same comment: Let's not run these tests then and address this bug separately. See thread: https://greatexpectationslabs.slack.com/archives/C04ST5TSR71/p1683147406394659